src/ui/layout/responsive/accessibility.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Accessibility for responsive apps
+description: >
+  xxx
+short-title: Accessibility
+---
+
+{% include docs/yt_shims.liquid %}
```

Comment:
**issue:** The `yt_shims.liquid` file was removed. That's [one error you're seeing](https://github.com/flutter/website/actions/runs/8332123719/job/22800541877?pr=10305#step:3:822).